src/repr/src/row.rs

```diff
@@ -402,10 +415,142 @@ mod columnation {
     }
 }
 
+mod columnar {
+    use crate::{Row, RowRef};
+    use columnar::{
+        AsBytes, Clear, Columnar, Container, FromBytes, HeapSize, Index, IndexAs, Len, Push,
+    };
+    use mz_ore::cast::CastFrom;
+
+    #[derive(Copy, Clone, Debug, Default, PartialEq, serde::Serialize, serde::Deserialize)]
+    pub struct Rows<BC = Vec<u64>, VC = Vec<u8>> {
```

Comment:
I was confused why `BC` isn't a container of `usize`, since that would be more natural for an index type. It's because there is no `Container` impl for `Vec<usize>`, right? Why is that?

Reply:
The issue is that we'd like to have WASM interoperability in the future, and `usize` in WASM is `u32`. To avoid the size differences, we use `u64` to represent `usize` instead.